source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -14,3 +19,72 @@
 .wporg-filtered-search-form {
 	display: block;
 }
+
+.sidebar-container .is-link-to-top {
+	display: none;
+}
+
+// Slot the search & table of contents into a floating sidebar on large screens.
+@media (min-width: 1200px) {
+	.sidebar-container {
+		// Size of remaining space in the wide center column.
+		--local--size-diff: calc(var(--wp--style--global--wide-size) - var(--wp--style--global--content-size));
+		// This is a little bit of a magic number, the max space between content & sidebar.
+		--local--content-gap: 145px;
+
+		position: absolute;
+		top: calc(var(--wp-global-header-offset, 0px) + var(--wp-local-header-offset, 0px));
+		// Right offset should be "edge spacing" at minimum, otherwise calculate it to be centered.
+		right: max(var(--wp--preset--spacing--edge-space), calc((100% - var(--wp--style--global--wide-size)) / 2));
+		width: calc(var(--local--size-diff) - var(--local--content-gap));
+		margin-top: var(--wp--preset--spacing--edge-space) !important;
+
+		&.is-fixed-sidebar {
+			position: fixed;
+		}
+
+		&.is-bottom-sidebar {
+			position: absolute;
+		}
+
+		&.is-fixed-sidebar .is-link-to-top,
+		&.is-bottom-sidebar .is-link-to-top {
+			display: block;
+		}
+	}
+}
+
+.wp-block-search.wp-block-search__button-inside .wp-block-search__inside-wrapper {
```

Comment:
This is also being added in #164 
https://github.com/WordPress/wporg-developer/pull/164/files#diff-5a967de023463491bebbfaca0909be4be783855fcace73f96a9a0e9c8060224aR18